workflow.rst

```diff
@@ -0,0 +1,51 @@
+Workflow
+========
+
+A workflow is a model of a process in your application. It may be the process
+of how a blog post goes from draft, review and publish. Another example is when
+a user submits a series of different forms to complete a task. Such processes are
+best kept away from your models and should be defined in configuration.
+
+A **definition** of a workflow consist of places and actions how to get from one
+place to another. The actions are called **transistions**. A workflow does also
+need to know each object's position in the workflow. That **marking store** writes
+to a property of object to remember the current place.
+
+.. note::
+
+    The terminology above is commonly used when discussing workflows and
+    `Petri nets`_
+
+The workflow component does also support state machines. A state machine is a subset
```

Comment:
It says it does support state machines and it has an article about it, isn't it?